### PR TITLE
feat: Add support for `CONFIG_CPU_FREQ_STAT`

### PR DESCRIPTION
**Pull request by rexagod**
Extracts data from the following files:
* `total_trans`
* `trans_table`
* `time_in_state`

Refer: https://www.kernel.org/doc/html/latest/cpu-freq/cpufreq-stats.html#configuring-cpufreq-stats.

Fixes: #428

### CODE REVIEW

File: sysfs/system_cpu.go [hunk 1]
```diff
@@ -292,19 +296,93 @@ func parseCpufreqCpuinfo(cpuPath string) (*SystemCPUCpufreqStats, error) {
 		}
 	}
 
+	// "total_trans" is the total number of times the CPU has changed frequency.
+	var cpuinfoFrequencyTransitionsTotal *uint64 = nil
+	cpuinfoFrequencyTransitionsTotalUint, err := util.ReadUintFromFile(filepath.Join(cpuPath, "stats", "total_trans"))
+	if err != nil {
+		if !(os.IsNotExist(err) || os.IsPermission(err)) {
+			return &SystemCPUCpufreqStats{}, err
+		}
```
Comment (discordianfish) [inline sysfs/system_cpu.go:305]: Why are you not returning the error here but continue silently?

Comment (rexagod) [reply sysfs/system_cpu.go:305]: https://github.com/prometheus/procfs/blob/9eed74490628b5d93360bd5e5971480c03fb43e0/sysfs/system_cpu.go#L273-L275

For these cases I saw that we were skipping over silently in other parts of the codebase, and kept the norm here.

Comment (discordianfish) [reply sysfs/system_cpu.go:305]: Ah I see, makes sense

**Review by discordianfish — APPROVED** [review]
LGTM

**Pull request merged**